code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -617,6 +618,11 @@ MATCHBOXES ARE ALSO IN FANCY.DM
 	light_color = LIGHT_COLOR_FIRE
 	var/lit = 0
 
+/obj/item/weapon/lighter/New()
+	..()
+	spawn(0)
```

Comment:
Because in random/New() I set the color_suffix var before after calling the parent. It seemed like a good idea, in retrospect I should just call the parent after that. Let me do that.